### DISCUSSION TITLE
How do I use the Winged light collector in the libdukun140.so

### DISCUSSION THREAD

**Discussion by REMIRATATOULLIE** [thatskymod/Sky-CotL-Scripts#57] [Q&A]
Hi really I just cant figure out how to use the winged light collector. I usually used the one from libsmite but it started to crash my game. I figured people from here would know how it works so please help 🙏

**Comment by thatskymod** [comment]
Possible duplicate of #50.

Jump to [resolution](https://github.com/thatskymod/Sky-CotL-Scripts/discussions/50#discussioncomment-7307972).
